SpatialGDK/Source/SpatialGDKFunctionalTests/SpatialGDK/SpatialTestCharacterMovement/SpatialTestCharacterMovement.cpp

```diff
@@ -106,8 +106,9 @@ void ASpatialTestCharacterMovement::PrepareTest()
 			ATestMovementCharacter* PlayerCharacter = Cast<ATestMovementCharacter>(PlayerController->GetPawn());
 
 			// Since the character is simulating gravity, it will drop from the original position close to (0, 0, 40), depending on the size
-			// of the CapsuleComponent in the TestMovementCharacter
-			return IsValid(PlayerCharacter) && PlayerCharacter->GetActorLocation().Equals(FVector(0.0f, 0.0f, 40.0f), 2.0f);
+			// of the CapsuleComponent in the TestMovementCharacter. However, depending on physics is not good for tests, so I'm
```

Comment:
Would be even better if maybe we just have more control over whether and where it drops. Maybe turn off gravity for it?

Reply:
I didn't want to mess around more with this test, I don't have the concept of why it is the way it is, so just tried to keep it as close as possible but making it work.

Reply:
@mironec I encourage you to create a follow up ticket for this test improvement, but I agree with @nafonso that this improvement is out of scope of this fix.